drivers/mrf24j40/mrf24j40_internal.c

```diff
@@ -87,6 +87,11 @@ void mrf24j40_init(mrf24j40_t *dev)
     mrf24j40_reg_write_short(dev, MRF24J40_REG_BBREG2, MRF25J40_BBREG2_CCAMODE1 );
     mrf24j40_reg_write_short(dev, MRF24J40_REG_CCAEDTH, 0x60);
     mrf24j40_reg_write_short(dev, MRF24J40_REG_BBREG6, MRF24J40_BBREG6_RSSIMODE2 );
+#if MRF24J40_USE_EXT_PA_LNA
```

Comment:
Adding a
```
    mrf24j40_reg_write_short(dev, MRF24J40_REG_TRISGPIO, 0x08);
    mrf24j40_reg_write_short(dev, MRF24J40_REG_GPIO, 0x08);
```
didn't hurt the MRF24J40MD and if it helps the MRF24J40MC we might as well add it as well add it unconditionally.
I did not test if it negatively affects power consumption on the D module though (where the GPIO should be handled automatically)

Reply:
Thank you for your comments. I'll check all of this as soon as possible (this weekend I hope as I don't have too much time at work). I don't know if you can also participate to this PR ?